### ISSUE TITLE
Proposal to contributing in Pytorch version of this implementation

### ISSUE THREAD

**Issue by GARV-k** [alirezatwk/Convolutional-Neural-Network-Based-Image-Watermarking-using-Discrete-Wavelet-Transform#9]
Hello authors. I am currently working on building a pytorch version of this project. 
However I would actually like to know some details about your training. 

Can you please report the loss value at the end of 60 epochs (according to batches of 10) and the time it took for you to train the pipeline.
